Jenkinsfile.trigger

```
@@ -126,55 +133,31 @@ node {
 						}
 					}
 				}
-			}
-			// we're done triggering GHA, so we're completely done with this job
-			breakEarly = true
-			return
-		}
-	}
-}
-
-if (breakEarly) { return } // thanks Jenkins...
-
-// now that we have our parsed queue, we can release the node we're holding up (since we handle GHA builds above)
-def pastFailedJobs = readJSON(text: pastFailedJobsJson)
-def newFailedJobs = [:]
-
-for (buildObj in queue) {
-	def identifier = buildObj.source.arches[buildObj.build.arch].tags[0]
-	def json = writeJSON(json: buildObj, returnText: true)
-	withEnv([
-		'json=' + json,
-	]) {
-		stage(identifier) {
-			echo(json) // for debugging/data purposes
-
-			def res = build(
-				job: 'build-' + env.BASHBREW_ARCH,
-				parameters: [
-					string(name: 'buildId', value: buildObj.buildId),
-				],
-				propagate: false,
-				quietPeriod: 5, // seconds
-			)
-			// TODO do something useful with "res.result" (especially "res.result != 'SUCCESS'")
-			echo(res.result)
-			if (res.result != 'SUCCESS') {
-				def c = 1
-				if (pastFailedJobs[buildObj.buildId]) {
-					// TODO more defensive access of .count? (it is created just below, so it should be safe)
-					c += pastFailedJobs[buildObj.buildId].count
+			} else {
+				def res = build(
+					job: 'build-' + env.BASHBREW_ARCH,
+					parameters: [
+						string(name: 'buildId', value: buildObj.buildId),
+					],
+					propagate: false,
+					quietPeriod: 5, // seconds
+				)
+				// TODO do something useful with "res.result" (especially "res.result != 'SUCCESS'")
+				if (res.result != 'SUCCESS') {
+					def c = 1
+					if (pastFailedJobs[buildObj.buildId]) {
+						// TODO more defensive access of .count? (it is created just below, so it should be safe)
+						c += pastFailedJobs[buildObj.buildId].count
+					}
+					// TODO maybe implement some amount of backoff? keep first url/endTime?
+					newFailedJobs[buildObj.buildId] = [
+						count: c,
+						identifier: identifier,
+						url: res.absoluteUrl,
+						endTime: (res.startTimeInMillis + res.duration) / 1000.0, // convert to seconds
+					]
+					error()
```

Since we got rid of the `echo`, we could stuff `res.result` in here (because otherwise it's an exception without a message, which is kind of silly but we don't really have any more information here than "the job failed" :joy:)

```suggestion
					error(res.result)
```